.github/CI_README.md

```diff
@@ -1,5 +1,13 @@
 GitHub CI for the IC Repo
 
+## Using custom CI labels
+Note that setting custom CI logic via the pull request title has been deprecated and we now use labels instead. See labels below for custom logic that can be enabled:
+
+`CI_ALL_BAZEL_TARGETS`: runs all bazel targets and uploads them to s3. \
+`CI_OVERRIDE_DIDC_CHECK`: skips the backwards compatibility didc check.
+
+Adding a label alone will not trigger CI, you will need to retrigger it by either opening & closing the PR or adding an empty commit with `git commit -m 'retrigger CI' --allow-empty`.
```

Comment:
Would it make sense to rerun the tests if the labels chanced? I think we can use the `pull_request` `labeled` activity type: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request

though we would need to make sure that e.g. the automatic label update (which adds `gix`, etc) does not trigger a CI run.

That'll avoid polluting the PR with lots of empty commits

Reply:
We have 2 workflows that add labels so this would be tricky if feasible at all. We can keep it simple and add info that user needs to push new/empty commit after he adds a label.

Reply:
I looked into this and I don't think it's feasible. We don't want to trigger CI on labels added or removed, because PRs are automatically labeled with the team when they are marked as ready. GitHub doesn't provide any way to trigger on specific labels (unless you get into really messy if statements).

It's not perfect, but I think it's good enough. And we also didn't re-trigger PRs when the title changed.